tensorflow/tools/pip_package/BUILD

```diff
@@ -64,7 +65,9 @@ transitive_hdrs(
         "//tensorflow/python/framework:python_op_gen",
         "//tensorflow/python/client:tf_session_helper",
         "//third_party/eigen3",
-    ] + if_cuda([
+    ] + if_rocm([
```

Comment:
Suggestion: maybe we shouldn't gate this for ROCm only.  These headers would likely benefit CUDA as well.

Reply:
My thinking here is that might delay the upstream merge, but I can remove that